src/mlpack/methods/ann/layer/batch_norm_impl.hpp

```diff
@@ -24,6 +25,8 @@ template<typename InputDataType, typename OutputDataType>
 BatchNorm<InputDataType, OutputDataType>::BatchNorm() :
     size(0),
     eps(1e-8),
+    average(true),
+    momentum(0.1),
```

Comment:
since average is `true` we should initialize momentum to 0?

Reply:
Sure, makes sense.